cost-analyzer/templates/cost-analyzer-checks-template.yaml

```diff
@@ -21,12 +21,7 @@ spec:
             - node
             - ./node/cron.js
             resources:
-              requests:
-                cpu: "20m"
-                memory: "100Mi"
-              limits:
-                cpu: "100m"
-                memory: "200Mi"
+{{ toYaml .Values.kubecostFrontend.resources | indent 14 }}
```

Comment:
This should be .Values.kubecostChecks.resources , right?

Reply:
Oops, my bad